dask/tests/test_base.py

```diff
@@ -963,6 +1170,48 @@ def test_persist_array_bag():
     assert list(b) == list(bb)
 
 
+@pytest.mark.skipif("not db")
+def test_persist_bag():
+    a = db.from_sequence([1, 2, 3], npartitions=2).map(lambda x: x * 2)
+    assert len(a.__dask_graph__()) == 4
+    b = a.persist(scheduler="sync")
+    assert isinstance(b, db.Bag)
+    assert len(b.__dask_graph__()) == 2
+    assert b.compute(scheduler="sync") == a.compute(scheduler="sync")
```

Comment:
There's also a `dask.bag.utils.assert_eq` utility to use here and in the few tests that follow

Reply:
fixed